### ISSUE TITLE
UIWhiteColor() is not at the center of the picker.

### ISSUE THREAD

**Issue by hunterbryant** [johankasperi/SwiftHSVColorPicker#2]
After setting the initial selectedColor to UIWhileColor(), the indicator isn't actually at the center of the picker, which it should be.  For some reason white is not the center color of the picker.  The white area is towards the blue hue area.
<img width="375" alt="img1" src="https://cloud.githubusercontent.com/assets/5898326/12072671/09a27962-b0a9-11e5-82b6-436b72ccc57a.png">

**Comment by hunterbryant** [comment]
Ok so I figured out that the reason it is shifted up and to the left is because when creating the actual color wheel, the frame of it is CGRectMake(20, 20, self.frame.width-40, self.frame.height-40).  I think that this means in pointAtHueSaturation the x and y dimensions are shifted 20pts from the margins.  I found what I think is a solution by adding 20pts to the x and y in the function.

```
let x = dimension / 2 + radius * cos(hue * CGFloat(M_PI) * 2) + 20
let y = dimension / 2 + radius * sin(hue * CGFloat(M_PI) * 2) + 20
```

I believe that this solves this issue throughout the class, but I haven't looked into it too in depth.
